typescript/packages/well-log-viewer/src/WellLogViewer.tsx

```diff
@@ -247,10 +251,13 @@ export default class WellLogViewer extends Component<
     ): void {
         if (
             this.props.welllog !== prevProps.welllog ||
+            this.props.wellLogSets !== prevProps.wellLogSets ||
```

Comment:
this.props.wellLogSets ?? this.props.welllog !== prevProps.wellLogSets ?? prevProps.welllog